source/actions3/BioXAS/BioXASMAXvMotorMoveToLimitActionInfo.cpp

```diff
@@ -0,0 +1,67 @@
+#include "BioXASMAXvMotorMoveToLimitActionInfo.h"
+
+BioXASMAXvMotorMoveToLimitActionInfo::BioXASMAXvMotorMoveToLimitActionInfo(const AMControlInfo &motor, CLSMAXvMotor::Limit limitSetpoint, QObject *parent) :
+	AMActionInfo3(QString(), QString(), QString(), parent)
```

Comment:
Both of those field are updated in `updateDescriptionText()`, called below. Is that sufficient?